scripts/fork/helpers/calculateDrawTimestamps.ts

```diff
@@ -0,0 +1,8 @@
+export function calculateDrawTimestamps(prizeTier: any, draw: any) {
+  const endTimestampOffset = prizeTier.endTimestampOffset;
+  const startTimestampOffset = draw.beaconPeriodSeconds;
+  const startTime = draw.timestamp - startTimestampOffset;
+  const endTime = draw.timestamp - endTimestampOffset;
```

Comment:
```suggestion
  const startTime = draw.timestamp - draw.beaconPeriodSeconds;
  const endTime = draw.timestamp - prizeTier.endTimestampOffset;
```
I still don't understand why we use `draw.beaconPeriodSeconds` for `startTimestampOffset` and not `draw.startTimestampOffset`.